src/main/java/seedu/address/model/person/ClientStatus.java

```diff
@@ -35,25 +35,55 @@ public String toString() {
 
     public final Status value;
 
+    /**
+     * Constructs a {@code ClientStatus}.
+     * Only to be used when reading Json files and by other methods in the {@code ClientStatus} class.
+     *
+     * @param status The ordinal of the status enumeration.
+     */
     public ClientStatus(int status) {
         value = Status.values()[status];
     }
+
+    /**
+     * Returns a {@code ClientStatus} for non-client persons.
+     * Used only when adding a non-client person to the addressbook.
+     */
     public static ClientStatus initNotClientStatus() {
         return new ClientStatus(0);
     }
 
+    /**
+     * Returns a {@code ClientStatus} for client persons.
+     * Used only when adding a client person to the addressbook.
+     */
     public static ClientStatus initClientStatus() {
         return new ClientStatus(1);
     }
```

Comment:
Good use of ClientStatus constructor to distinguish between partners and clients.